docs/iot/TOC.md

```diff
@@ -0,0 +1,34 @@
+# [IoT and Tizen IoT](/iot/overview.md)
+
+# Getting Started
+## [Tizen IoT](/iot/getting-started/overview.md)
+## [Installing Tizen Studio](/iot/getting-started/tizen-studio-install.htm)
```

Comment:
From here, the filename extensions should be changed from `htm` to `md`.

Reply:
done